src/Symfony/Component/Finder/README.md

```diff
@@ -42,11 +42,11 @@ foreach ($finder->in('s3://bucket-name') as $file) {
 Resources
 ---------
 
-You can run the unit tests with the following command:
-
-    $ cd path/to/Symfony/Component/Finder/
-    $ composer install
-    $ phpunit
+  * [Documentation](https://symfony.com/doc/current/components/finder.html)
```

Comment:
We should make sure that all these docs are actually present in the official doc though (and opening issues if something is missing in the official doc)